lib/core-integration/src/Test/Integration/Framework/DSL.hs

```diff
@@ -1191,6 +1215,23 @@ for = flip map
 --
 -- Helper for delegation statuses
 --
+getSlotParams
+    :: (Context t)
+    -> IO (EpochNo, SlotParameters)
+getSlotParams ctx = do
+    r2 <- request @ApiNetworkInformation ctx
```

Comment:
why `r2` ;-) and not `r1`

Reply:
I think that you actually wrote this, I just moved it to different place ;)

Reply:
done in d45edd7 - refactor param names in getSlotParams